forc-plugins/forc-publish/src/tarball.rs

```diff
@@ -156,6 +151,8 @@ mod test {
 
         // Run the function
         let result = create_tarball_from_current_dir(&temp_output_dir);
+        result.unwrap();
```

Comment:
redundant call to the same function?

Reply:
Fixed in [1bdcba4](https://github.com/FuelLabs/sway/pull/7253/commits/1bdcba4084831f8004423022aef19d63a8cfd1bf), thanks for catching this!